src/views/virtualmachines/actions/tests/useVirtualMachineActionsProvider.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { cleanup } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+
+import useVirtualMachineActionsProvider from '../hooks/useVirtualMachineActionsProvider';
+
+import { exampleRunningVirtualMachine } from './mocks';
+
+jest.mock('@openshift-console/dynamic-plugin-sdk', () => ({
+  useK8sWatchResource: jest.fn(() => [[], true]),
+}));
+afterEach(cleanup);
+
+describe('useVirtualMachineActionsProvider tests', () => {
+  it('Test running VM actions', () => {
+    const { result } = renderHook(() =>
+      useVirtualMachineActionsProvider(exampleRunningVirtualMachine),
+    );
+    expect(result.current).toMatchSnapshot();
+
+    const actions = [...result.current.actions];
+    // Running vm should have stop action
+    const stopAction = actions.find((a) => a.id === 'vm-action-stop');
+    expect(stopAction).toBeTruthy();
+  });
+
+  it('Test stopped VM actions', () => {
+    const stoppedVM: V1VirtualMachine = {
+      ...exampleRunningVirtualMachine,
+      status: { printableStatus: 'Stopped' },
+    };
+    const { result } = renderHook(() => useVirtualMachineActionsProvider(stoppedVM));
+    expect(result.current).toMatchSnapshot();
+
+    const actions = [...result.current.actions];
+    // Stopped vm should have start action
+    const startAction = actions.find((a) => a.id === 'vm-action-start');
```

Comment:
no need for `[...result.current.actions]`
just write `result.current.actions.find((a) => a.id === 'vm-action-start');`

Reply:
I was thinking maybe to change it to something else so it would check if all actions are list something like:
`it('Test running VM actions', () => {
    const { result } = renderHook(() =>
      useVirtualMachineActionsProvider(exampleRunningVirtualMachine),
    );

    const actions = [...result.current.actions].map((action) => !action.disabled && action.id);

    // Running vm should have stop,restart,pause,migrate and delete actions
    expect(actions).toEqual([
      'vm-action-stop',
      'vm-action-restart',
      'vm-action-pause',
      'vm-action-migrate',
      'vm-action-delete',
    ]);`
    
    @glekner wdyt?